DOCS/man/options.rst

```diff
@@ -899,9 +899,9 @@ Program Behavior
         on whether most of your URLs need youtube-dl parsing.
 
     ``exclude=<URL1|URL2|...``
-        A ``|``-separated list of URL patterns which mpv should not use with
-        youtube-dl. The patterns are matched after the ``http(s)://`` part of
-        the URL.
+        A ``|``-separated list of URL patterns which mpv should not try parsing
+        with youtube-dl first when ``try_ytdl_first`` is ``yes``. The patterns
+        are matched after the ``http(s)://`` part of the URL.
```

Comment:
Why this is updated? The logic didn't change? This should exclude use of ytdl for those url, regardless of `try_ytdl_first`

Reply:
Seems I misunderstood how it works.